src/thorin/be/llvm/amdgpu.cpp

```diff
@@ -76,4 +76,8 @@ llvm::Value* AMDGPUCodeGen::emit_reserve(llvm::IRBuilder<>& irbuilder, const Con
     return emit_reserve_shared(irbuilder, continuation, true);
 }
 
+llvm::Value* AMDGPUCodeGen::emit_local_memory(llvm::IRBuilder<>& irbuilder, const Continuation* continuation) {
+    return emit_local_memory_base_ptr(irbuilder, continuation);
+}
```

Comment:
why is this identical function copy/pasted here and in nvvm ? since there are no alternative implementations there's no reason to do this, just have the default impl in llvm.cpp be this one

Reply:
I just did it the same way it was done for `reserve_shared()`. It's not valid to use this function unless one of the GPU backends is used. So the base implementation in `llvm.cpp` emits an error, and the GPU implementations override it with a call to the actual implementation.